src/sidebar/components/test/annotation-share-control-test.js

```diff
@@ -195,13 +199,38 @@ describe('AnnotationShareControl', () => {
       const wrapper = createComponent({ isPrivate: testcase.isPrivate });
       openElement(wrapper);
 
-      const permissionsEl = wrapper.find(
-        '.annotation-share-panel__permissions'
-      );
+      const permissionsEl = wrapper.find('.annotation-share-panel__details');
       assert.equal(permissionsEl.text(), testcase.expected);
     });
   });
 
+  it('renders an explanation if annotation cannot be shared in context', () => {
```

Comment:
This test matches the implementation, although I queried whether perhaps we should show these links, just sharing the single annotation page rather than the incontext link.